docs/source/configuration/in-memory-caching.mdx

```diff
@@ -63,6 +61,8 @@ supergraph:
     warmed_up_queries: 100
 ```
 
+(In addition, the router can use the contents of the [persisted query list](./persisted-queries) to prewarm the cache. By default, it does this when loading a new schema but not on startup; you can [configure](./persisted-queries#persisted-queries#experimental_prewarm_query_plan_cache) it to change either of these defaults.)
```

Comment:
I think I might have gotten this wrong from the slack convo, but aren't we changing the default to be "warm PQs at startup" with an additional option to "warm PQs at schema reload"?

Reply:
(although the option's *name* is changing, with migration)